attributes/default.rb

```diff
@@ -1,5 +1,5 @@
 # osquery version and packs.
-default['osquery']['version'] = '2.3.2'
+default['osquery']['version'] = '2.4.0'
```

Comment:
I'd like to leave the interpolation outside the scope of the `version` variable to make overriding clearer